packages/edit-site/src/components/header/document-actions/index.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { Button, Dropdown } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { DOWN } from '@wordpress/keycodes';
+
+export default function DocumentActions( {
+	primaryText,
+	secondaryText,
+	isSecondaryItemActive = false,
+} ) {
+	return (
+		<div className="edit-site-document-actions">
+			{ primaryText ? (
+				<Dropdown
```

Comment:
I'm not sure where we've landed on this being a dropdown; I think it makes a lot of sense for the document/template label to open some settings, but I'm not so sure about the template part label. In some designs the template part label could be useful as a way to select the template part itself. This could confuse the interactions between dropdown and selection.

For now, I'd suggest we move forward with _just_ the labels as indicators of what template and template part of being edited.

Reply:
> In some designs the template part label could be useful as a way to select the template part itself.

If the label already shows the name of the template part, is it not already selected or hovered?  And in the case of being hovered you couldn't hover it and click the name in the top-bar at the same time so... I guess I dont understand how clicking the name of the Template Part here would act to select it?

Reply:
> I'm not sure where we've landed on this being a dropdown; I think it makes a lot of sense for the document/template label to open some settings

So what should it open for controlling the settings then?  I thought the current design mocks for these settings were a dropdown?

Reply:
It might be best to focus on showing the template part being edited first — and then add the interaction in another PR. I'll defer to @dubielzyk here.

Reply:
that sounds fine to me. I used the dropdown because I thought the document settings menu would be activated on click of this "area," and based on current designs that does at least sort of look like a dropdown menu 😅 

Totally on board with keeping just the labels for the time being.

Reply:
>In some designs the template part label could be useful as a way to select the template part itself. This could confuse the interactions between dropdown and selection.

I think we should use the same treatment for both. The behavior should be similar between template and page/post. If we agree that there will be settings in there at some point, I'd say we should use the dropdown interaction.

If we use a staggered approach I think we could push it out in phases:
Phase 1: Show label. Top label is template or page/post. Bottom label is template part.
Phase 2: Add dropdown interaction. Add renaming of Template or Page/Post name. [Mocks](
![image](https://user-images.githubusercontent.com/2896062/93277004-73b1c380-f7c1-11ea-9b2b-dfff58d17a74.png))
Phase 3: Start looking at adding relevant settings

How does that sound to everyone? @shaunandrews @noahtallen @Addison-Stavlo

Reply:
yep! Per:

> Top label is template or page/post

I think this opens a can of worms conceptually, because if we go with "page," we're starting to think about the site editor from a user's page-first perspective rather than a technical person's template-first perspective. To be clear, it's easy to change the variable rendering this technically :D


I'm mostly convinced by the argument to think about the site editor from a page-first perspective.

Reply:
I 100% agree that we should go page/post first. I think we only show the template if they use an entry-point that is about editing a specific template.

Reply:
>  I think we only show the template 

right, and to be clear, the template would get "shown" in the editor either way, but it would be more of the focus there

Reply:
I mean this:
![image](https://user-images.githubusercontent.com/2896062/93285443-20497080-f7d5-11ea-88a4-fff45876491f.png)

So yeah, we'd show a way to see what template is being affected either way, but if a users wants to edit their _About_ page, then we'd display About in the top bar. If they are going to editing a specific template, we'd show that in the top bar. Hope that makes more sense 😄

Reply:
That does make sense. I think, as we discussed in the meeting, this is tricky because there are no entrypoints currently which let us infer if the user wants to mainly edit a template or a page. :)